src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -589,71 +589,71 @@ void M()
         {
             await TestInClassAsync(
 @"$$Int32 i;",
-                MainDescription("struct System.Int32"));
+                MainDescription("int"));
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.QuickInfo)]
         public async Task TestBuiltInInt()
         {
             await TestInClassAsync(
 @"$$int i;",
-                MainDescription("struct System.Int32"));
+                MainDescription("int"));
```

Comment:
❗️ This particular change concerns me enough that I think we should stop to really think through more cases. If I hover over a built-in type, I would expect it to show the true underlying type name regardless of my particular preference for built-in or full type names.